pkg/monitor/monitor/util.go

```diff
@@ -742,6 +728,18 @@ func getMonitorVolumes(config *core.ConfigMap, monitor *v1alpha1.TidbMonitor, tc
 		volumes = append(volumes, monitor.Spec.AdditionalVolumes...)
 	}
 
+	// add asset tls
```

Comment:
ditto

Reply:
I think we can take the `tls-assets` secret as default volume.Not only prometheus use it save secret,other container will  
also need ,for example thanos etc.